src/System.Security.Cryptography.Csp/src/Resources/Strings.resx

```diff
@@ -145,6 +145,9 @@
   <data name="Cryptography_OpenInvalidHandle" xml:space="preserve">
     <value>Cannot open an invalid handle.</value>
   </data>
+  <data name="Cryptography_Operation_Not_Supported" xml:space="preserve">
```

Comment:
Cryptography_CAPI_Required, perhaps?  "Operation Not Supported" is a bit generic, and might get accidentally used for the wrong thing later.